docs/user/alerting/rule-types.asciidoc

```diff
@@ -2,7 +2,7 @@
 [[rule-types]]
 == Rule types
 
-A rule is a set of <<alerting-concepts-conditions, conditions>>, <<alerting-concepts-scheduling, schedules>>, and <<alerting-concepts-actions, actions>> that enable notifications. {kib} provides two types of rules:  rules specific to the Elastic Stack and rules specific to a domain.
+A rule is a set of <<alerting-concepts-conditions, conditions>>, <<alerting-concepts-scheduling, schedules>>, and <<alerting-concepts-actions, actions>> that enable notifications. {kib} provides rules built into the Elastic Stack and rules registered by one of the {kib} apps.
```

Comment:
Instead of talking about the UI in each section, I think it would be better to add this paragraph after the intro paragraph:

You can create most rules types in [Stack Management > Rules and Connectors](https://www.elastic.co/guide/en/kibana/current/create-and-manage-rules.html). For information on creating security rules, refer to [Create a detection rule](https://www.elastic.co/guide/en/security/current/rules-ui-create.html).